packages/react/src/lib/accessibility/Behaviors/Toolbar/toolbarBehavior.ts

```diff
@@ -22,6 +23,7 @@ const toolbarBehavior: Accessibility = (props: any) => ({
       shouldFocusInnerElementWhenReceivedFocus: true,
     },
   },
+  childrenBehavior: props => toolbarButtonBehavior(props),
```

Comment:
I think I tried this or something similar, and I would get an error in `renderComponent` because it will look for a function to be called with props and I am passing that function already called.

or am I missing something?

Reply:
hmm, let me try this